### ISSUE TITLE
Shorter emoji alias(s)

### ISSUE THREAD

**Issue by kyle-rader** [hachyderm/community#54]
## 📝 Context 
There is a short (`24`) char limit to Display Names.

Mine for example: `Kyle :QueerCat_Bisexual: :rust`

## 🤔 Issue 
I enjoy outing myself via emoji in my display name as being both Bi and a Rustacean :). But alas, `QueerCat_Bisexual` takes too many chars and I'm 1 short. 

## 🎊 Proposed Solution 
Emoji Alias: `:queercat_bi:` would do the trick. Perhaps others might recommend or find other useful shortened names so we may be revel in emoji display names 🎉.

**Comment by kyle-rader** [comment]
Also thanks for redirecting me to "issue tracker" on hachyderm :)

**Comment by quintessence** [comment]
Going going to comment here and note the other two raised a similar issue :) I'm going to look into this, the tooling right now doesn't support "mass renaming" so we're likely going to need to rename the ones mostly commonly used in people's display names. A couple things to note:

* Part of the reason that emoji are named they way they are is inherited from imported emoji sets :)
* The name of the emoji is what's read by a screenreader

For the latter, that means that (easy example) "Jayne Cobb : k8s :" is read as "Jayne Cobb colon k eight s colon". The more emojis you add to your display name, the longer that bit is for a screenreader to read. This also means that when emoji names are shortened, we need to do so in a way that is still accessible to a screenreader. From this view, `queercat_bi` would still be fine, but `kubernetes` is more accessible than `k8s`.

TLDR we're looking at this, there're just a few things we need to consider to implement :)

**Comment by quintessence** [comment]
Going to close this - note we still have it open in our internal tracker to see how we can do this in an accessible, scaled way :)

**Comment by kyle-rader** [comment]
@quintessence Thanks for the update! I love the focus on scale and inclusion and how this could impact different scenarios :)
